app/code/Magento/Theme/Block/Html/Topmenu.php

```diff
@@ -246,7 +246,7 @@ protected function _getHtml(
             }
 
             $html .= '<li ' . $this->_getRenderedMenuItemAttributes($child) . '>';
-            $html .= '<a href="' . $child->getUrl() . '" ' . $outermostClassCode . '><span>' . $this->escapeHtml(
+            $html .= '<a href="' . $child->getUrl() . '" ' . $outermostClassCode . ' role="menuitem"><span>' . $this->escapeHtml(
```

Comment:
```suggestion
            $html .= '<a href="' . $child->getUrl() . '" '
            . $outermostClassCode
            . ' role="menuitem"><span>'
            . $this->escapeHtml(
```

To fix coding standard:

FILE: /var/www/html/app/code/Magento/Theme/Block/Html/Topmenu.php
-------------------------------------------------- --------------------
FOUND 0 ERRORS AND 1 WARNING AFFECTING 1 LINE
-------------------------------------------------- --------------------
 249 | WARNING | Line exceeds 120 characters; contains 129 characters